src/pages/index.js

```diff
@@ -161,8 +161,8 @@ function handleDeleteSubmit(evt) {
       selectedCard.remove();
       closeModal(deleteModal);
     })
-    .catch(console.error)
-    .finally((submitButton.textContent = "Delete"));
+    .catch(() => console.error)
```

Comment:
` .catch(console.error);` should be used instead of `.catch(() => console.error)`.  


It should be either `.catch(() => console.error());` or ` .catch(console.error);`, but not  `.catch(() => console.error)` because the `console.error` is not called inside the arrow function, so it doen't work


**Please, correct it everywhere in the code**